android/app/src/main/AndroidManifest.xml

```diff
@@ -7,11 +7,14 @@
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
     <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" android:maxSdkVersion="28"/>
```

Comment:
why you remove `android:maxSdkVersion="28"`?

Reply:
this  line is outdated i returned it  in the final version